CMakeLists.txt

```diff
@@ -198,6 +198,11 @@ if(WANT_ENV_FLAGS)
 	set(CMAKE_BUILD_TYPE Undefined)
 	set(CMAKE_STATUS_DEBUG_SUPPORT "Environment-driven")
 else()
+	if(MSVC)
+		# QT4 has /Zc:wchar_t-, QT5 has /Zc:wchar_t
+		set(CMAKE_CXX_FLAGS "/MP /Zc:wchar_t /EHsc /source-charset:utf-8")
+		set(CMAKE_C_FLAGS "/MP /Zc:wchar_t /EHsc /source-charset:utf-8")
+	endif()
 	if(WANT_DEBUG)
```

Comment:
I don't agree with concept of WANT_DEBUG here at all, because in MSVC Release vs Debug is selected after CMake is done. But that's irrelevant for this PR...